pkg/gemspec/parse_testcase.go

```diff
@@ -0,0 +1,11 @@
+package gemspec
+
+import "github.com/aquasecurity/go-dep-parser/pkg/types"
+
+var (
+	GemspecNormal = []types.Library{
+		{"async", "1.25.0"},
+	}
+
+	GemspecMalformed = []types.Library{}
+)
```

Comment:
You don't have to put them in a separate file when the expected value is small enough.